### PR TITLE
Remove Dojo leftovers from previous optimization

### PR DESCRIPTION
**Pull request by ylavoie**
This replaces PR#3205

### CODE REVIEW

File: lib/LedgerSMB/Template.pm [hunk 1]
```diff
@@ -410,7 +410,7 @@ sub new_UI {
             if $LedgerSMB::App_State::Company_Config;
     my $UI_vars = {
         dojo_theme => $dojo_theme // $LedgerSMB::Sysconfig::dojo_theme,
-        dojo_built => $LedgerSMB::Sysconfig::dojo_built,
```
Comment (sbts) [inline lib/LedgerSMB/Template.pm]: I'd like to leave dojo_built available to js if possible

Comment (ylavoie) [reply lib/LedgerSMB/Template.pm]: Sorry, we agreed on that and I forgot.

File: old/lib/LedgerSMB/Form.pm [hunk 2]
```diff
@@ -599,8 +598,8 @@ qq|<meta http-equiv="content-type" content="text/html; charset=$self->{charset}"
        }
        var lsmbConfig = {dateformat: '$dformat'};
     </script>
-    <script type="text/javascript" language="JavaScript" src="$dojo_location/dojo/dojo.js"></script>
-    <script type="text/javascript" language="JavaScript" src="$dojo_location/lsmb/main.js"></script>
+    <script type="text/javascript" language="JavaScript" src="$LedgerSMB::Sysconfig::dojo_location/dojo/dojo.js"></script>
+    <script type="text/javascript" language="JavaScript" src="$LedgerSMB::Sysconfig::dojo_location/lsmb/main.js"></script>
```
Comment (ehuelsmann) [inline old/lib/LedgerSMB/Form.pm]: What's wrong with using `$dojo_location` here? We didn't change anything in the diff which would require the change.

Comment (ylavoie) [reply old/lib/LedgerSMB/Form.pm]: $dojo_location has been removed by this pr

**Pull request merged**